roles/engineblock/tasks/main.yml

```diff
@@ -64,6 +64,30 @@
   when:
     customlogo.stat.exists
 
+- name: Check if we have a custom background back image for the feedback page
+  local_action: stat path="{{ inventory_dir }}/files/background-back.svg"
+  register: customfeedbackbackground
```

Comment:
Could you prefix customfeedbackground with eb_ ?

Reply:
:+1: